components/bl-spoiler-effect/src/index.js

```diff
@@ -0,0 +1,298 @@
+import { useCallback, useEffect, useRef, useState } from 'react';
+
+const { cn } = BackendlessUI.CSSUtils;
+
+const EffectTypes = {
+  BLUR      : 'blur',
+  MORPHOLOGY: 'morphology',
+  MORPHOSE  : 'morphose',
+  MUSIC     : 'music',
+  RIPPLES   : 'ripples',
+  SMOKE     : 'smoke',
+  SNOW      : 'snow',
+  TURBULENCE: 'turbulence',
+};
+const EffectViews = {
+  [EffectTypes.BLUR]      : BlurEffect,
+  [EffectTypes.MORPHOLOGY]: MorphologyEffect,
+  [EffectTypes.MORPHOSE]  : MorphoseEffect,
+  [EffectTypes.MUSIC]     : MusicEffect,
+  [EffectTypes.RIPPLES]   : RipplesEffect,
+  [EffectTypes.SMOKE]     : SmokeEffect,
+  [EffectTypes.SNOW]      : SnowEffect,
+  [EffectTypes.TURBULENCE]: TurbulenceEffect,
```

Comment:
all effect components in separated file `effects` and rename them without `Effect`